models/base_model.py

```diff
@@ -16,13 +16,19 @@ class BaseModel():
         updated_at (Date): current datetime when an instance is updated
     """
 
-    def __init__(self):
+    def __init__(self, *args, **kwargs):
         """
-        Method call when an instance is created
+        Intialize object class with 2 arguments/parameters
         """
-        self.id = str(uuid.uuid4())
-        self.created_at = datetime.now()
-        self.updated_at = datetime.now()
+        if len(args) > 0:
+            for key in args[0]:
+                setattr(self, key, args[0][key])
+        else:
+            self.id = str(uuid.uuid4())
+            self.created_at = datetime.now()
+            self.updated_at = datetime.now()
+        for key in kwargs:
+            print("kwargs: {}: {}".format(key, kwargs[key]))
```

Comment:
Moni, you need to set the attributes instead of printing them. I mean you can check whether the key in kwargs is updated_at, created_at or id and set the with the value. else you need to add the key and value to the self.__dict__ obj.

Note: Remeber to convert date_string to datetime obj before to set the attribute. You can use: classmethod datetime.strptime(date_string, format)
More info: https://docs.python.org/3/library/datetime.html#datetime.datetime